docs/platforms/unity/user-feedback/index.mdx

```diff
@@ -15,18 +15,11 @@ User Feedback for **[ASP.NET](/platforms/dotnet/guides/aspnet/user-feedback/#int
 
 </Alert>
 
-You can create a form to collect the user input in your preferred framework, and use the SDK's API to send the information to Sentry. You can also use the widget, as described below. If you'd prefer an alternative to the widget or do not have a JavaScript frontend, you can use this API:
+You can create a form to collect the user input in your preferred framework, and use the SDK's API to send the information to Sentry. You can also use the widget, as described below. If you'd prefer an alternative to the widget or do not have a web frontend, you can use this API:
 
 ```csharp {tabTitle:C#}
 var eventId = SentrySdk.CaptureMessage("An event that will receive user feedback.");
 
-SentrySdk.CaptureUserFeedback(eventId, "user@example.com", "It broke.", "The User");
+SentrySdk.CaptureFeedback(eventId, "user@example.com", "It broke.", "The User");
```

Comment:
```suggestion
SentrySdk.CaptureFeedback("It broke.", "user@example.com", "The User", associatedEventId: eventId);
```

Since the EventID is optional in the new API, the order of the parameters has been changed (optional parameters come last). See [new API](https://github.com/getsentry/sentry-dotnet/blob/4821fe7068b91f8314cd8989c761cc28a8df37e5/src/Sentry/SentrySdk.cs#L503-L507):
> public static void CaptureFeedback(string message, string? contactEmail = null, string? name = null, string? replayId = null, string? url = null, SentryId? associatedEventId = null, Scope? scope = null, SentryHint? hint = null)